Products/zms/ZMSTextformat.py

```diff
@@ -99,8 +99,8 @@ def br_quote(text, subtag, REQUEST):
     
     if subtag == 'br':
       rtn += line+qcr
-      if c < len( ts):
-        rtn += '<%s />'%subtag
+      # if c < len( ts):

```

Comment:
The blockformat-manager ZMI added always a br-tag if definied as subtag to the preview rendering. Actually this interferes with after-pseudoclasses, so  I removed that (not needed?) suffix.